sql/core/pom.xml

```diff
@@ -128,6 +128,10 @@
       <artifactId>xbean-asm5-shaded</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.spark-project.hive</groupId>
+      <artifactId>hive-exec</artifactId>
+    </dependency>
```

Comment:
Why do we need to add this dependency to spark-core? The ORC data source lives in spark-hive. Can we move the newly added `OrcRecordReader.java` to spark-hive and remove this dependency?